.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,19 @@
-Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-XXX
+Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-XXXXX
 
 ## Checklist
 
 I have:
 
-- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
-- [ ] Made sure my feature is working on the relevant real / virtual devices.
-- [ ] Written **unit tests** for my feature.
-- [ ] Written **documentation on Notion** for my feature.
-- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
+- [ ] Made sure the title of my PR follows the convention `[PC-XXXXX] <summary>`.
+- [ ] Made sure my feature is working on the relevant real / virtual devices (mobile and web).
+- [ ] Written **unit tests** native and web for my feature.
 - [ ] Added a **screenshot** for UI tickets.
-- [ ] Attached a **ticket number** for any added TODO/FIXME \
-       (for tech tasks, give the best context about the TODO resolution: what/who/when).
+- [ ] Written **documentation on Notion** for my feature.
```

Comment:
tu peux supprimer je pense

Reply:
Pour `Written **documentation on Notion** for my feature.` Marion veut le laisser pour éviter les cas où il n’y a qu’un sachant sur le sujet qui n’écrit ou ne met pas à jour la doc (cf les liens). 

Donc à voir avec elle si c'est la bonne place pour ce genre de rappel ? Car c'est très très peu utilisé, même par les personnes qui sont supposé l'utiliser.

Reply:
@marionnousvalentin

Reply:
on peut suppr, ça sert à rien